static/helpers.js

```diff
@@ -15,36 +15,62 @@
     return Handlebars.helpers.each(data, options)
   }
   global.docxStyle = function (options) {
-    return new Handlebars.SafeString(`<docxStyle id="${options.hash.id}" textColor="${options.hash.textColor}" />`)
+    return new Handlebars.SafeString(
+      `<docxStyle id="${options.hash.id}" textColor="${options.hash.textColor}" />`
+    )
   }
 
   global.docxImage = function (options) {
-    if (!options.hash.src) {
-      throw new Error('docxImage helper requires src parameter to be set')
+    if (!options.hash.src && !options.hash.url) {

```

Comment:
We would prefer having just `src` param and don't use extra  `url` param. The same as the `img` html tag has.
You can pass to the src data uri like `src='data:image/png;base64,...'` but also `src='https://foo.com/img'`. You would just need to deal with recognizing what is data uri and what you should request.

Reply:
good idea, makes sense